packages/scheduler-prisma/src/helpers.ts

```diff
@@ -308,17 +308,17 @@ export const timestampHelperFactory =
     task: TaskCallback<T, R>
   ) =>
   async (args: T, context: JobContext): Promise<any> => {
-    const jobId = args?.context?.job?.id ?? context?.job?.id;
+    const jobId = args.context?.job?.id ?? context.job?.id;
 
     if (!jobId) {
       try {
-        return task(args, context);
+        return await task(args, context);
       } catch (e) {
         return null;
```

Comment:
Appreciate this has been committed (and wasn't your change), but super against any try/catch that swallows the error. It can be valid to do in some rare circumstances, but either way it should always log something to let us know it happened

Reply:
Very true. I was trying to stay away from any significant changes, but I should have added a log here.